src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryConfig.java

```diff
@@ -163,15 +163,27 @@
     @AttributeDefinition(name = "Allowed Vanity Path Location",
         description ="This setting can contain a list of path prefixes, e.g. /libs/, /content/. If " +
                     "such a list is configured, only vanity paths from resources starting with this prefix " +
-                    " are considered. If the list is empty, all vanity paths are used.")
+                    " are considered. If the list is empty, we fallback to resource.resolver.vanitypath.allowlist.")
```

Comment:
I think the fallback should be in the opposite direction: if the new property is not used, the old one will be used